lib/onegini.dart

```diff
@@ -41,17 +42,22 @@ class Onegini {
     OneginiEventListener eventListener, {
     String? securityControllerClassName,
     String? configModelClassName,
-    List<String>? twoStepCustomIdentityProviderIds,
+    List<Map<String, Object>>? customIdentityProviderConfigs,
     int? connectionTimeout,
     int? readTimeout,
   }) async {
     _eventListener = eventListener;
     try {
+      var customIdentityProviderConfigsJson = null;
+      if (customIdentityProviderConfigs != null) {
+        customIdentityProviderConfigsJson = [for (var x in customIdentityProviderConfigs) json.encode(x)];

```

Comment:
don't use `x`, use a name which describes what it contains

Reply:
done